xml/cap_depl_eks.xml

```diff
@@ -370,151 +288,347 @@ Custom TCP Rule    TCP          4443            0.0.0.0/0       CAP WSS
+       namespace, and the rest are in the <literal>scf</literal> namespace.
+   </para>
+         <para>
+         After deployment your domain must be changed to the load 
+         balancer IP address. The following table lists the required domain and 
+         sub-domains, using <literal>example.com</literal> as the example domain:
+     </para>
+  <informaltable>
+      <tgroup cols="2">
+          <thead>
+              <row>
+                  <entry>Domains</entry>
+                  <entry>Services</entry>
+              </row>
+          </thead>
+          <tbody>
+              <row>
+                  <entry>example.com</entry>
+                  <entry>scf/router-public</entry>
+               </row>
+               <row>
+                  <entry>*.example.com</entry>
+                  <entry>scf/router-public</entry>
+               </row>
+               <row>
+                  <entry>uaa.example.com</entry>
+                  <entry>uaa/uaa-public</entry>
+               </row>
+               <row>
+                  <entry>*.uaa.example.com</entry>
+                  <entry>uaa/uaa-public</entry>
+               </row>                
+              <row>
+                  <entry>tcp.example.com</entry>
+                  <entry>scf/tcp-router-public</entry>
+               </row>
+               <row>
+                  <entry>ssh.example.com</entry>
+                  <entry>scf/diego-access-public</entry>
+               </row>
+           </tbody>
+       </tgroup>
+   </informaltable>
+</sect1>
+ 
+<sect1 xml:id="sec.cap.eks-configuration">
+    <title>Deployment Configuration</title>
   <para>
    Use this example <filename>scf-config-values.yaml</filename> as a template
    for your configuration.
   </para>
 
 <screen>env:
-  # Domain for SCF. DNS for *.DOMAIN must point to a kube node's (not master)
-  # external IP address.
   DOMAIN: <replaceable>public_ip_of_node-VM.example.com</replaceable>
-
-  #### The UAA hostname is hardcoded to uaa.$DOMAIN, so shouldn't be
-  #### specified when deploying
-  # UAA host/port that SCF will talk to. If you have a custom UAA
-  # provide its host and port here.
   UAA_HOST: uaa.<replaceable>public_ip_of_node-VM.example.com</replaceable>
   UAA_PORT: 2793
  
   GARDEN_ROOTFS_DRIVER: overlay-xfs
   GARDEN_APPARMOR_PROFILE: ""
-
-sizing:
-  cc_uploader:
-    capabilities: ["SYS_RESOURCE"]
-  diego_api:
-    capabilities: ["SYS_RESOURCE"]
-  diego_brain:
-    capabilities: ["SYS_RESOURCE"]
-  diego_ssh:
-    capabilities: ["SYS_RESOURCE"]
-  nats:
-    capabilities: ["SYS_RESOURCE"]
-  router:
-    capabilities: ["SYS_RESOURCE"]
-  routing_api:
-    capabilities: ["SYS_RESOURCE"]
-
+  
+services:
+  loadbalanced: true
+  
 kube:
-  # The IP address assigned to the kube node pointed to by the domain.
-  #### the external_ip setting changed to accept a list of IPs, and was
-  #### renamed to external_ips
   external_ips:
   - <replaceable>private_ip_of_node-VM</replaceable>
 
```

Comment:
We should perhaps take this out. external_ips is even more confusing when we are using automatic LB set up. @troytop ?

Reply:
:heavy_check_mark: